modules/calendar/calendar.php

```diff
@@ -1836,7 +1841,13 @@ public function fix_post_date_on_update_part_two( $post_ID, $post_after, $post_b
 		$wpdb->update( $wpdb->posts, array( 'post_date' => $post_date ), array( 'ID' => $post_ID ) );
 		clean_post_cache( $post_ID );
 	}
-	
+
+
+	public function is_calendar_view() {
+		global $pagenow;
+
+		return ( 'index.php' === $pagenow && isset( $_GET['page'] ) && $_GET['page'] === 'calendar' );
```

Comment:
Can this be switched to a yoda conditional?

`'calendar' === $_GET['page']`

Reply:
Same as https://github.com/Automattic/Edit-Flow/pull/441/files#r240276539 :)